lib/Dialects/LinalgTransform/Transforms/TrackingCSE.cpp

```diff
@@ -7,7 +7,7 @@
 //===----------------------------------------------------------------------===//
 
 #include "Dialects/LinalgTransform/TrackingCSE.h"
-#include "TrackingListener.h"
+#include "../IR/TrackingListener.h"
```

Comment:
This feels annoying, any include_dirs you can add to the cmake?

Reply:
I can just move the header.